### DISCUSSION TITLE
Your IP is blocked by BingAi

### DISCUSSION THREAD

**Discussion by owlcloud** [waylaidwanderer/node-chatgpt-api#416] [Help]
I have an error everywhere that the IP from bingai is blocked. on replits with and without proxy. on your own machine with and without a proxy. What to do?

**Comment by hibobmaster** [comment]
Provide userToken or cookies in settings.js can solve the problem.
https://github.com/waylaidwanderer/node-chatgpt-api/blob/654ae3ccef46b3fa6f53cca6c91757f9a09a4137/settings.example.js#L44-L47

**Reply by HUAYIXIAO** [reply]
It doesnt work😥

**Comment by yuwangi** [comment]
Suggest deploying to foreign servers or transitioning through Cloudflare Worker

**Reply by yuwangi** [reply]
refer to  this https://zhile.io/2023/12/24/gemini-pro-proxy.html

**Reply by evengit** [reply]
试用了一下 确实可以  感谢~
